test/acceptance/docker-compose.yaml

```diff
@@ -7,12 +7,12 @@ services:
     depends_on:
       - selenium
       - hub
-      - browserstack
-
+      # - browserstack
     environment:
       TEST_ENVIRONMENT_ROOT_URL: ${TEST_ENVIRONMENT_ROOT_URL}
       CHROME_ARGS: "--headless --ignore-certificate-errors"
-          
+      BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY}
+
```

Comment:
Adding the key to this image so it can be used by the BrowserStack local npm package